src/stores/spaces/SpaceStore.ts

```diff
@@ -162,6 +164,14 @@ export class SpaceStoreClass extends AsyncStoreWithClient<EmptyObject> {
         SettingsStore.monitorSetting("feature_dynamic_room_predecessors", null);
     }
 
+    /**
+     * Returns a promise that resolves when the space store is ready.
+     * This happens after an initial hierarchy of spaces and rooms has been computed.
+     */
+    public get isReady(): Promise<void> {
```

Comment:
`isReady` sounds like it returns a boolean, the naming does not describe a promise which resolves when ready, could you pick a better name?

Reply:
https://github.com/element-hq/element-web/pull/29399/commits/02cf7a69b853b4b4c9c9191a77a3a1702bc33ddd